src/platform/tf-node.ts

```diff
@@ -10,4 +10,5 @@
 
 import * as tf from '@tensorflow/tfjs-node';
 export const requireTf = async () => tf;
+export {tf};
 
```

Comment:
Does the above statically link it? As long as we only import `tf-web` on demand, then that would lead to this only being imported on demand as well I believe. If the design requires that this also be done on demand for some reason (I haven't reviewed service register implementation yet) that would be good to know.

Reply:
`tf-web` is statically linked.